packages/core/src/builder.class.ts

```diff
@@ -29,9 +29,14 @@ const urlParser = {
 
     // IE 11 pathname handling workaround
     // (IE omits preceeding '/', unlike other browsers)
-    if (out.pathname && typeof out.pathname === 'string' && out.pathname.indexOf('/') !== 0) {
```

Comment:
so if `out.pathname` is empty string, then the conditional would be false and this wouldn't add the `/`